### ISSUE TITLE
Raidboss pop ups/sounds arent working, timers are however

### ISSUE THREAD

**Issue by hyrulezelda** [quisquous/cactbot#815]
Since completely deleting/reinstalling act and cactbot with the ngold plugin I haven't been able to see pop ups nor hear the sound effects for raidboss mechanics in any fight. The timers still show up and are perfect per fight. Just the raidboss sounds and pop ups aren't. 

I went to Summerford Farms and started a countdown to see if it still worked there and they did everything was perfectly fine with both timers and pop ups and sounds, but for some reason in any of the actual raids in the game sounds and pop up messages don't work.

**Comment by quisquous** [comment]
Hmm! Are there any errors messages or anything in your overlayplugin log when this happens?

**Comment by bubbabri** [comment]
Raidboss timers and callouts are not working for myself as well. I deleted the entire cactbot folder and unzipped a fresh copy from the release page for 0.15.1. Noticed that the dps meter was not showing, but got that fixed after re-adding it. Upon adding a MiniParseOverlay for raidboss, it gives me the following errors:

`[1/15/2020 9:51:53 PM] Error: Got a subscription for missing event "onUserFileChanged"!
[1/15/2020 9:51:53 PM] Error: Got a subscription for missing event "onForceReload"!
[1/15/2020 9:51:53 PM] Error: JS Handler call failed: System.Exception: Tried to call missing handler "cactbotLoadUser"!
   at RainbowMage.OverlayPlugin.EventDispatcher.CallHandler(JObject e)
   at RainbowMage.OverlayPlugin.EventDispatcher.ProcessHandlerMessage(IEventReceiver receiver, String data)
[1/15/2020 9:51:53 PM] Error: JS Handler call failed: System.Exception: Tried to call missing handler "cactbotLoadData"!
   at RainbowMage.OverlayPlugin.EventDispatcher.CallHandler(JObject e)
   at RainbowMage.OverlayPlugin.EventDispatcher.ProcessHandlerMessage(IEventReceiver receiver, String data)
[1/15/2020 9:51:53 PM] Info: raid boss: BrowserConsole: Uncaught (in promise) TypeError: Cannot read property 'detail' of null (Source: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/ngld/cactbot/resources/user_config.js, Line: 35)
`

The 3 DLL versions are below:
OriginalFilename: FFXIV_ACT_Plugin.dll
FileVersion: 2.0.4.13

OriginalFilename: OverlayPlugin.dll
FileVersion: 0.13.2.0

OriginalFilename: CactbotOverlay.dll
FileVersion: 0.15.1.0

ACT is version 3.4.5 Release 266.
